modules/apps/analytics/analytics-client-js/src/analytics.js

```diff
@@ -193,15 +193,9 @@ class Analytics {
 		};
 
 		const storedIdentityHash = storage.get(STORAGE_KEY_IDENTITY_HASH);
-		let newIdentityHash = hash(bodyData);
+		const newIdentityHash = hash(bodyData);
```

Comment:
Now that the browser fingerprint is not used, does this hash become more likely to collide?